core/src/main/java/nu/yona/server/subscriptions/service/UserUpdateService.java

```diff
@@ -109,18 +109,25 @@ public void requestOverwriteUserConfirmationCode(String mobileNumber)
 
 	private void setOverwriteUserConfirmationCodeIfNotDoneRecently(String mobileNumber, User user)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This method has 21 lines, which is greater than the 20 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS138)